WcaOnRails/app/assets/stylesheets/registrations.scss

```diff
@@ -25,3 +25,16 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
```

Comment:
Confusingly enough, registrations.scss is for competition registrations, not WCA account registrations. users.scss is probably the best place for this.